Sources/XCHammer/XcodeTarget.swift

```diff
@@ -1213,6 +1260,17 @@ public class XcodeTarget: Hashable, Equatable {
     }
 
     func extractProductType() -> ProductType? {
+        // TODO: add a way to indicate to to the generator to filter these
+        if self.type == "objc_library" && self.label.value.hasSuffix("_objc")  {
```

Comment:
This is the kind of thing we could handle with the `tags` approach you mentioned right?